### ISSUE TITLE
Place Markers

### ISSUE THREAD

**Issue by DonnC** [arconsis/measurements#37]
Hello, im glad i found this package, good work 
Will it be possible to place a marker on a given point, since i can measure distance and pointers are placed, will it be possible if user has ability to maybe tap somewhere and i place a marker on that location

I hope its clear, looking forward to your response

**Comment by c15yi** [comment]
Hey @DonnC thanks for the feedback.

What do you mean by placing markers? Do you want to "highlight" a point that was set for a measurement?

Or do you want to get the coordinate of the point on the background?

**Comment by DonnC** [comment]
i understand on measuring the distance, the plugin places those points and lines right which is great, but can i place a Marker (point) programmatically at a certain coordinate on the floor plan?

**Comment by DonnC** [comment]
@ChristianNavolskyi my idea is centered around indoor navigation, so i will do my measurements on the physical ground to get the distance where the user is, and i will map those coordinates to my floor plan. From there i would like to use this plugin to place those mapped coordinates or point as a marker in the app

is something like this possible because i just need a way to place a dot or marker on the floor plan somewhere in the app. This is because the plugin already places points for measurements when i tap on the floor plan on  the screen, will it be possible to expose such a functionality to place points to where ever i need to place them on the floor plan on the app

**Comment by DonnC** [comment]
Let me rephrase, is there a way to expose the methods that translate app screen coordinates to the  floor plan measurement system and vice versa. I want to be able to place a dot using floor plan measurement values programmatically then the app should place a dot on the screen where that measurement value is. 
This is a request of the reverse process of what the plugin does, and is it possible to do so @ChristianNavolskyi because i want to put a marker on the screen using code and not when user taps on the screen first

**Comment by c15yi** [comment]
Sorry for the late reply. After checking out the code (haven't been working on it for some time) I see that we currently only have the way out, as you already found out.

Currently the other way is not implemented, but would of course be a nice feature.

**Comment by DonnC** [comment]
Okay thank you, are there any current plans for it or that you plan to?
 indeed i have seen its been a long time since you have worked on it, i would appreciate it any future plans you have

**Comment by c15yi** [comment]
Yes this project has been sleeping a bit in the last months.

Initially we wanted to see if there is demand for such plugin at all and then decide on further steps.

Now that you are here I am quite happy to see it come to good use. I will try to find some time to work on it.
The plugin also needs some updating of its dependencies, since a lot has changed in the flutter world lately.

**Comment by DonnC** [comment]
Indeed, thank you so much for the assurance, will look forward to it and hopefully also write about this plugin.
